### PR TITLE
cache_bug

### PR DESCRIPTION
**Pull request by dephillipsmichael**
Added call to sign out with completion when a user taps "forget passcode"

### CODE REVIEW

File: BridgeAppSDK/SBAAppDelegate.swift [hunk 1]
```diff
@@ -656,7 +656,8 @@ public let SBAMainStoryboardName = "Main"
     
     private func resetPasscode() {
         
-        // reset the user
+        // reset the user and sign them out of bridge        
+        SBABridgeManager.signOut(cmpletion: nil)
```
Comment (syoung-smallwisdom) [inline BridgeAppSDK/SBAAppDelegate.swift]: cmpletion -> completion

File: BridgeAppSDK/SBABridgeManager.h [hunk 2]
```diff
@@ -120,6 +120,11 @@ typedef void (^SBABridgeManagerCompletionBlock)(id _Nullable responseObject, NSE
     completion:(SBABridgeManagerCompletionBlock _Nullable)completionBlock;
 
 /*!
+ Sign out user on the bridge server
+ */
++ (void)signOutWithCmpletion:(SBABridgeManagerCompletionBlock _Nullable)completionBlock;
```
Comment (syoung-smallwisdom) [inline BridgeAppSDK/SBABridgeManager.h]: signOutWithCmpletion -> signOutWithCompletion

**Review by syoung-smallwisdom — CHANGES_REQUESTED** [review]
Typos. Otherwise, looks good.

File: BridgeAppSDK/SBABridgeManager.h [hunk 3]
```diff
@@ -120,6 +120,11 @@ typedef void (^SBABridgeManagerCompletionBlock)(id _Nullable responseObject, NSE
     completion:(SBABridgeManagerCompletionBlock _Nullable)completionBlock;
 
 /*!
+ Sign out user on the bridge server
```
Comment (Erin-Mounts) [inline BridgeAppSDK/SBABridgeManager.h]: missing '.' at the end, Bridge should be capitalized, and should document the completion parameter (see above).

File: BridgeAppSDK/SBABridgeManager.m [hunk 4]
```diff
@@ -160,6 +160,19 @@ + (void)signIn:(NSString *)email password:(NSString *)password completion:(SBABr
     }];
 }
 
++ (void)signOutWithCmpletion:(SBABridgeManagerCompletionBlock _Nullable)completionBlock {
+    [SBBComponent(SBBAuthManager) signOutWithCompletion:^(NSURLSessionTask *task, id responseObject, NSError *error) {
+#if DEBUG
+        if (error != nil) {
+            NSLog(@"Error with signup: %@", error);
```
Comment (Erin-Mounts) [inline BridgeAppSDK/SBABridgeManager.m]: should say "Error with signOut: ..."

**Review by Erin-Mounts — CHANGES_REQUESTED** [review]
Couple of nits re: doc comments and debug messages.

**Comment by dephillipsmichael** [comment]
@syoung-smallwisdom @Erin-Mounts my fault, fixed the issues.

**Review by Erin-Mounts — APPROVED** [review]

**Review by syoung-smallwisdom — APPROVED** [review]

**Pull request merged**